helm/prometheus-rules/templates/alerting-rules/capo.rules.yml

```diff
@@ -49,4 +49,4 @@ spec:
             description: |-
               {{`OpenStackCluster {{$labels.exported_namespace}}/{{$labels.name}} failure reason is {{$labels.reason}}.`}}
             opsrecipe: remove-errors-from-capi-capo-crs/
-{{- end }}
+{{- end }}
```

Comment:
Why removing this EOF?

Reply:
happens due rebasing -- reverted the removal